data_structures/trie/radix_tree.py

```diff
@@ -191,39 +198,42 @@ def print_tree(self, height: int = 0) -> None:
             value.print_tree(height + 1)
 
 
-def test_trie() -> bool:
-    words = "banana bananas bandana band apple all beast".split()
-    root = RadixNode()
-    root.insert_many(words)
-
-    assert all(root.find(word) for word in words)
-    assert not root.find("bandanas")
-    assert not root.find("apps")
-    root.delete("all")
-    assert not root.find("all")
-    root.delete("banana")
-    assert not root.find("banana")
-    assert root.find("bananas")
+## write unit test for the code using unittest library with
+## logic similar to test_trie() function
+## and call it from main()
 
-    return True
 
+class TestRadixNode(unittest.TestCase):
```

Comment:
This boilerplate adds unnecessary complexity.

Reply:
Sure Will Remove it. But its the best way to do this